source/_components/mqtt_room.markdown

```diff
@@ -72,5 +72,6 @@ Example JSON that should be published to the room topics:
 This component works with any software that is sending data in the given format. Each client should post the discovered devices in its own subtopic of the configured topic.
 Instead of developing your own application, you can also use any of these already existing clients:
 
-- [**room-assistant**](https://github.com/mKeRix/room-assistant): looks for Bluetooth LE beacons, based on Node.js
+- [**room-assistant**](https://github.com/mKeRix/room-assistant): runs on a raspberry pi, and looks for Bluetooth LE beacons, based on Node.js
```

Comment:
This is a bit weird since it might suggest it only runs on a Raspberry Pi?

Reply:
Good point. Removed change in subsequent commit.